### PR TITLE
Enhancement/pager teams

### PR DESCRIPTION
**Pull request by jspaleta**
This PR  implements feature request: #38 

Note:
using prefix instead of suffix as  prefix works better with available single charactger shorthand cmdline arguments.

All logic constrained as part of checkArgs function
team token will be preferred over auth token if team token is available in matching environment variable.

Added initial test of new logic.
Will be adding more tests now.

### CODE REVIEW

**Comment by jspaleta** [comment]
reverted back to suffix... removed the short argument option for both team-name.
Which is fine.. most of the time you'll be using the team name from the annotation support.

File: README.md [hunk 1]
```diff
@@ -212,6 +215,63 @@ metadata:
...
 
+### Pager teams
+
+Instead of specifying the authentication token directly in the check or agent annotations, you can instead reference a pager team name, which will then be used to lookup the corresponding token from the handler environment. 
+Corresponding pager team token environment variables can be populated in the handler environment in 3 different ways
+1. Explicitly set in the handler definition
+2. Kept as Sensu [secrets][13] and referenced in the handler definition
+3. Defined in the [backend service environment file][14], read in at backend service start.
+
+Pager team names will be automatically suffixed with configured --pagerduty-team-suffix (default: `_pagerduty_suffix`)
```
Comment (echlebek) [inline README.md]: --pagerduty-team-suffix doesn't square with the app's flags

**Review by echlebek — COMMENTED** [review]
I'd recommend making the naming more consistent between the flags and environment variables. For instance it's `--pager-team` and `PAGERDUTY_TEAM`, kind of makes it hard for people to remember. Other than that, looks good!

**Comment by jspaleta** [comment]
@echlebek  fixed in newest commit

File: README.md (hunk 1 above)
Comment (jspaleta) [reply README.md]: fixed in latest commit

**Review by echlebek — APPROVED** [review]

**Pull request merged**